runtime/wasmer.go

```diff
@@ -325,9 +325,13 @@ type Runtime struct {
 	trie *trie.Trie
```

Comment:
Add the license statement at the beginning for `runtime/types.go` and `runtime/wasmer.go`